src/arkit.cs

```diff
@@ -70,13 +73,17 @@ public enum ARHitTestResultType : nuint {
 		EstimatedHorizontalPlane = 1 << 1,
 		ExistingPlane = 1 << 3,
 		ExistingPlaneUsingExtent = 1 << 4,
+		[iOS (11,3)]
+		ExistingPlaneUsingGeometry = (1 << 5),
```

Comment:
Why the (1 << 5) and not simply 1 << 5?

Reply:
Because `sharpie` :P

I agree it's not needed, if I have to editor this PR again I'll fix it. It's minor though.